routes/todoRoute.js

```diff
@@ -0,0 +1,11 @@
+const router = require ('express').Router()
+const TodoController = require ('../controllers/todoController.js')
+
+router.get ("/", TodoController.findTodos)
+router.get ("/find/:id", TodoController.findTodoById)
```

Comment:
ga usah pake find, langsung aja /:id
```suggestion
router.get ("/:id", TodoController.findTodoById)
```